FreshRSS/FreshRSS
# Improve performance #5835

Closes #5835

Changes proposed in this pull request:

- 1s => 0s waiting


How to test the feature manually:

1. mark an article as read
2. see that it is faster now


Pull request checklist:
